### PR TITLE
Jim/more pima schema

### PR DESCRIPTION
**Pull request by jimduff-usds**
This PR adds a column that SimpleReport is sending us, and makes formatting changes that Pima requested.

## Changes
- Added a new mapper than can concatenate column's strings together.
- 

## Checklist
- [X] Tested locally?
- [ ] Added new dependencies?
- [ ] Updated the docs?
- [X] Added a test?
- [X] Did you check for sensitive data, and remove any?
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## Fixes 

- #138, #136

## To Be Done

- Add a separator param to the concat mapper.
- Did not do the specimen_site yet (the body part, eg 'Nose'), because unsure how to proceed with 25k SNOMED values.
- Commented out new Test_result_status value, because SimpleReport wasn't ready to send to us today.  Need to uncomment-this-out when they are ready.

### CODE REVIEW

File: prime-router/metadata/organizations-local.yml [hunk 1]
```diff
@@ -30,20 +30,20 @@
             filePath: ./upload
         format: CSV
 
-  - name: pa-phd
-    description: Pennsylvennia PHD
+  - name: pima-az-phd
+    description: Pima County, Arizona PHD
```
Comment (jimduff-usds) [inline prime-router/metadata/organizations-local.yml:34]: The diff is really weird, but what happened is I deleted "pa" and add "pima-az-phd" organizations.

File: prime-router/metadata/schemas/AZ/pima-az-covid-19-csv.schema [hunk 2]
```diff
@@ -26,25 +26,26 @@ elements:
...
     csvFields: [{name: Device_ID}]
 
-  # Add'l field per request from the County
+
   # TODO Badness: As items are added in prime-data-input-client/backend/src/main/resources/application.yaml,
   #  we need to add them here.
   - name: device_type
     type: CODE
     mapper: use(standard.equipment_model_id)
+    documentation:   "Additional field per request from the County.  HHS lists 4 codes for 'Prior test' question that may match whats desired here:  Molecular, Antigen, Antibody/Serology, and UNK."
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/AZ/pima-az-covid-19-csv.schema:48]: Using the new documentation field!

File: prime-router/metadata/schemas/AZ/pima-az-covid-19-csv.schema [hunk 3]
```diff
@@ -26,25 +26,26 @@ elements:
     type: TEXT
 
   - name: standard.test_performed
-    csvFields: [{name: Ordered_test_code}]
+    csvFields: [{name: Ordered_test_code, format: $display}]
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/AZ/pima-az-covid-19-csv.schema:29]: Pima requested readable strings for all the CODE values.

File: prime-router/metadata/schemas/AZ/pima-az-covid-19-csv.schema [hunk 4]
```diff
@@ -67,35 +68,39 @@ elements:
   - name: standard.testing_lab_specimen_id
     csvFields: [{name: Testing_lab_specimen_ID}]
 
+# Removed until SimpleReport sends this column
+#  - name: standard.test_result_status
+#    csvFields: [{name: Test_result_status, format: $display}]
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/AZ/pima-az-covid-19-csv.schema:73]: SimpleReport wasn't ready for this today.

File: prime-router/metadata/schemas/AZ/pima-az-covid-19-csv.schema [hunk 5]
```diff
@@ -67,35 +68,39 @@ elements:
...
+
   - name: standard.test_result
-    csvFields: [{name: Test_result_code}]
+    csvFields: [{name: Test_result_code, format: $display}]
 
   - name: standard.illness_onset_date
-    csvFields: [{name: Illness_onset_date}]
+    csvFields: [{name: Illness_onset_date, format: "yyyy-MM-dd"}]
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/AZ/pima-az-covid-19-csv.schema:79]: Sigh.

File: prime-router/metadata/schemas/AZ/pima-az-covid-19-csv.schema [hunk 6]
```diff
@@ -162,7 +167,9 @@ elements:
   - name: standard.testing_lab_county
     csvFields: [{name: Testing_lab_county}]
 
-  - name: standard.ordering_facility_name
+  # Nonstandard
+  - name: organization_and_facility
+    mapper: 'concat(organization_name, standard.ordering_facility_name)'
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/AZ/pima-az-covid-19-csv.schema:172]: Calling the new concat(str1, str2, strN) mapper.

File: prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema [hunk 7]
```diff
@@ -141,6 +145,11 @@ elements:
   - name: standard.testing_lab_county
     csvFields: [{name: Testing_lab_county}]
 
+  - name: organization_name
```
Comment (jimduff-usds) [inline prime-router/metadata/schemas/PrimeDataInput/pdi-covid-19.schema:148]: This was a request from SimpleReport - they want to break apart 'facility' and 'organization', so we added this new field.  Facility still uses 'ordering_facility_name'.

File: prime-router/metadata/valuesets/common.valuesets [hunk 8]
```diff
@@ -140,9 +140,9 @@
     - code: I
       display: No results available; specimen received, procedure incomplete
     - code: M
-      display: Corrected, not final		added v2.9
```
Comment (jimduff-usds) [inline prime-router/metadata/valuesets/common.valuesets:143]: typos

File: prime-router/src/main/kotlin/FakeReport.kt [hunk 9]
```diff
@@ -74,7 +74,7 @@ class FakeReport {
                         else -> TODO()
                     }
                 }
-                Element.Type.TELEPHONE -> faker.numerify("##########") // faker.phoneNumber().cellPhone()
+                Element.Type.TELEPHONE -> faker.numerify("##########:1:") // faker.phoneNumber().cellPhone()
```
Comment (jimduff-usds) [inline prime-router/src/main/kotlin/FakeReport.kt:77]: This fix got inadvertently left out

File: prime-router/src/main/kotlin/Mappers.kt [hunk 10]
```diff
@@ -62,6 +62,33 @@ class UseMapper : Mapper {
...
 
+/**
+ * The mapper concatenates a list of column values together.
+ * Call this like this:
+ * concat(standard.organization_name, standard.ordering_facility_name)
+ * @todo add a separator arg.
+ * @todo generalize this to call any kotlin string function?
+ */
+class ConcatenateMapper : Mapper {
+    override val name = "concat"
```
Comment (RickHawesUSDS) [inline prime-router/src/main/kotlin/Mappers.kt:73]: Nice. I've wanted this mapper.

**Review by RickHawesUSDS — APPROVED** [review]

**Pull request merged**